packages/frontend/src/App.svelte

```diff
@@ -62,6 +63,9 @@ onMount(() => {
       <Route path="/disk-images/build/:name/:tag" breadcrumb="Build" let:meta>
         <Build imageName={decodeURIComponent(meta.params.name)} imageTag={decodeURIComponent(meta.params.tag)} />
       </Route>
+      <Route path="/disk-images/createVM/:imageName/:imagePath" breadcrumb="Create Virtual Machine" let:meta>
```

Comment:
Updated the reference here @deboer-tim regarding removing 'macadam' reference and for it to be /createVM.